server/partials/footer.html

```diff
@@ -21,10 +21,20 @@ <h4 class="ui header">Community</h4>
       <div class="three wide column">
         <h4 class="ui header">Social</h4>
         <div class="ui link list">
-          <a class="item" href="https://twitter.com/fomanticui" target="_blank">
+          <a class="item twitter" href="https://twitter.com/fomanticui" target="_blank">
+            <svg xmlns="http://www.w3.org/2000/svg" x="0px" y="-20px" width="15" height="15" viewBox="0,0,256,256"
```

Comment:
What's the source of the svg icon? Please provide a link. We need to check the usage license.

Reply:
https://icons8.com/icons/set/twitter

Reply:
There is an official brand logo provided by x.com, we should use that instead

https://about.twitter.com/en/who-we-are/brand-toolkit
https://about.twitter.com/content/dam/about-twitter/x/brand-toolkit/x-logo.zip

Reply:
okay